flexmeasures/api/v3_0/sensors.py

```diff
@@ -65,12 +71,27 @@ class SensorAPI(FlaskView):
             "account": AccountIdField(
                 data_key="account_id", load_default=AccountIdField.load_current
             ),
+            "all_accessible": fields.Boolean(required=False, missing=False),
+            "page": fields.Int(
+                required=False, validate=validate.Range(min=1), default=1
+            ),
+            "per_page": fields.Int(
+                required=False, validate=validate.Range(min=1), default=10
+            ),
+            "filter": SearchFilterField(required=False, default=None),
         },
         location="query",
     )
     @permission_required_for_context("read", ctx_arg_name="account")
     @as_json
-    def index(self, account: Account):
+    def index(
+        self,
+        account: Account,
+        all_accessible: bool = False,
```

Comment:
Please update the docstring when the parameters are added and change the main interpretation of what is possible with the endpoint.

Reply:
I’ve added this now, but feel free to check if I might have missed anything.